src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -131,11 +131,12 @@ export const DetailsForm: React.FC<DetailsFormProps> = ({
       if (!user) return;
       try {
         // unfortunately the typing is off for react-hook-forms.
```

Comment:
[OOS] We can probably resolve that.. which would be better than explicitly `as` casting below, which is going to hide any potential typing issues we would otherwise run into/be protected from.